website/docs/learn/5-deploy-your-project.md

```diff
@@ -0,0 +1,106 @@
+    * Switch the project in the header bar to your new "dbt Tutorial" project.
+3. Complete the project setup flow:
+    * Connect to Snowflake using the same credentials you used to connect in the [Setting Up](tutorial/setting-up.md) instructions.
+    * Add a repository — choose the Github integration, and link to the repository you set up in the [Setting Up](tutorial/setting-up.md) instructions.
+
+
+## Create a deployment environment
+<LoomVideo id="bb6ea5b628ef4d019f9167f6ddf738cc" />
+
+1. Click the hamburger menu, and then `Home`
+2. Under "Deployment Environments" select "Create a new one"
+3. Name your deployment environment (e.g. "Production")
+4. Add a target schema (e.g. "analytics_[initialsurname]") — this is the schema that dbt will build into.
+
+<Alert type="info">
+We would typically name our deployment schema "analytics" (without a suffix) to tell our users that it's "the good place". For this tutorial, as we have a number of students using the same Snowflake account, we are using suffixes to differentiate between each student's work.
+</Alert>
+
+## Create and run a job
+_See above video_
+
+Jobs are a set of dbt commands (e.g. `dbt run`, `dbt test`) that you want to run on a schedule.
+
+1. After creating your deployment environment, you should be directed to the page for new environment. If not, select the hamburger menu, and then `Jobs`.
+2. Click `New Job` giving it a name (e.g. "Production run"), and linking it to the Environment you just created.
+3. Check the box for "Generate docs".
+4. Ensure you have the following commands as part of your job:
+      * `dbt run`
+      * `dbt test`
+5. **Do not** set a schedule for your project to run -- while your organization's project **should** run regularly, there's no need to run this project on a schedule, and doing so could result in using up your BigQuery credits.
+6. Select `Save`, and then `Run now` to run your job.
+7. Click into the run to see the progress — once the run is complete, click "View Documentation" to see the docs for your project.
+
+### FAQs
+<FAQ src="faqs/failed-prod-run" />
+
+## Next steps
+
+<Alert type="success">
+Congratulations 🎉! You've just deployed your first dbt project! <strong>Please reply to the email you received with your credentials with a link to your GitHub repository.</strong>
+</Alert>
+
+We're very excited to see you at Learn. In the meantime, here's a few suggestions if you want to get a headstart:
+
+### Start working on your own project
+Ready to get started with your own project? We recommend doing the tutorial a second time to create a new project. You should use your own warehouse, along with a query that you frequently run, to build your first models!
+
+### Learn some best practices
+Now that you've got a working dbt project, read more about some of our [best practices](https://docs.getdbt.com/docs/best-practices), or go back this tutorial and read some of the FAQs.
+
+### Keep building this project
```

Comment:
This touches on concepts/extensions we'll cover in dbt Learn, with the expectation that students may or may not be familiar. I think it's great if people are exposed to them sooner, I don't think we need to say anything to that effect here.